_posts/2016-09-10-stripe-dress.md

```diff
@@ -0,0 +1,44 @@
+author: neuroamanda
+image:
+  feature: 2016-09-10-stripe-dress.jpg
+  credit: Gareth Williams
+  caption: Horizontal-striped dress
+---
+
+_Horizontal stripes on clothing are in fact slimming_
+
+[The dress](https://en.wikipedia.org/wiki/The_dress) sparked the hottest fashion debate of the 21st century, but another mystery has occupied scientists and fashionistas alike since the mid-19th century: do horizontal stripes make you look fat?
+The answer may seem obvious, but experimental results and a well-documented illusion contradict most people's intuition that horizontal stripes make the wearer look wider.
+
+<!--break-->
+
+As pioneering vision scientist Helmholtz [noted back in 1867](http://poseidon.sunyopt.edu/helmholtz/OCRVolume3.pdf) (see page 206!), a person wearing horizontal stripes should, contrary to popular belief, appear _thinner_ than one with vertical stripes.
+This observation is the basis for the [Helmholtz illusion](http://www.bbc.co.uk/radio4/features/sywtbas/finalists/stripes/), at least as it applies to shapes, not bodies.
+A team of researchers from Japan tested whether the size of the body wearing stripes changes the strength of the effect.
+
+[In line with previous research](http://ipe.sagepub.com/content/2/1/69), they found that the common idea that horizontal stripes make one look fat is in fact wrong.
+In the experiment, a group of students rated computer-generated figures wearing either a horizontally or vertically striped shirt.
+The figures had different degrees of thin and fat body shapes to determine when the horizontal and vertical-striped figures looked ‘equally fat’ for each observer.
+This measurement is a baseline for each person’s perception, and all participants perceived a person with horizontal stripes to be thinner than a person with vertical stripes.
+This agrees with the Helmholtz illusion phenomenon, that a horizontally striped square is perceived as ‘thinner’ than one with vertical stripes.
+
+![Horizontal-striped dress]({{ site.url }}/images/2016-09-10-stripe-dress.jpg)
+
+> _Horizontal-striped dress. [Image](https://www.flickr.com/photos/gareth1953/9497924862) by Gareth Williams / [CC BY 2.0](https://creativecommons.org/licenses/by/2.0/)_
+
+With the experimental bodies, however, the effect was stronger for the thin figures than the fat ones.
+Thus, as the title of this research paper states, “the Helmholtz illusion makes you look fit only when you are already fit”.
+Whether the students were presented with the fat or thin figures first also made a difference: if fat figures were shown first, the Helmholtz effect was diminished, and for the fat figures the judgments actually bumped the baseline in the opposite direction.
+In this condition, the horizontal stripes on fat figures really were perceived to make the figure fatter.
+
+The main takeaway from this simple experiment is that context can have a large effect on how something is perceived.
```

Comment:
Comments noted. There was no discussion in the paper of socio-cultural contexts so can't really branch out to that in a summary of the paper. But this effect has been replicated elsewhere (well, England), so it's not just Japan.